src/main/scala/mesosphere/marathon/core/appinfo/AppInfoModule.scala

```diff
@@ -21,8 +21,11 @@ class AppInfoModule @Inject() (
     healthCheckManager: HealthCheckManager,
     marathonSchedulerService: MarathonSchedulerService,
     taskFailureRepository: TaskFailureRepository) {
+
+  val ec = NamedExecutionContext.fixedThreadPoolExecutionContext(8, "app-info-module")
```

Comment:
`ec`?   are we trying to stay within a linter line length rule :) 

could we make this more readable pls?

Reply:
There are a few akka/scala specific and accepted "short names" e.g. `ec` for `ExecutionContext`, `mat` for `Materializer` etc. Falls into the same category as  loop counter `for(i=0,..)`

Reply:
Should this thread pool be ideally related to the number of threads available?

Should we make it configurable? (even if via an undocumented environment variable)?

It seems like we should ideally create the execution context in the module and share it with the pods status controller?

Reply:
`ec` is fine for me for the reason @zen-dog pointed out.
I agree with @timcharper that the number of threads could as well be configurable.